specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/examples/BackupResourceEncryptionConfig_Get.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rishgrp",
+    "vaultName": "rishTestVault",
+    "api-version": "2021-01-01"
```

Comment:
>2021-01-01 [](start = 20, length = 10)

update api versions in example files too

Reply:
done